app/css/style.css

```diff
@@ -8498,49 +8498,111 @@ path.state:hover {
   right: 190px;
   top: 160px; }
 
+<<<<<<< HEAD
```

Comment:
this merge line shouldn't be in the commit, its a left over conflict that didn't get resolved